Documentation/Administration/Installation/LegacyInstallation.rst

```diff
@@ -47,6 +47,7 @@ Installing on a Unix Server
    .. code-block:: bash
       :caption: /var/www/site/$
 
+      mkdir public
       cd public
```

Comment:
```suggestion
      cd /path/to/webroot
```
How about using path to webroot instead?

Reply:
Then the caption above must be adjusted too:

`:caption: /var/www/site/$`

Reply:
it already is `:caption: /var/www/site/$`?

Reply:
Yup, so you say the installation is in /var/www/site/ and then you say, change the directory to /path/to/webroot, this is inconsistent. Anyway, this change is not worth the time. Close it therefore.